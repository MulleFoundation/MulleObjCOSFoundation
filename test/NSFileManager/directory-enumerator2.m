//
//  main.m
//  archiver-test
//
//  Created by Nat! on 19.04.16.
//  Copyright Â© 2016 Mulle kybernetiK. All rights reserved.
//

#ifdef __MULLE_OBJC__
# import <MulleObjCOSFoundation/MulleObjCOSFoundation.h>
#else
# import <Foundation/Foundation.h>
# pragma message( "Apple Foundation")
#endif


#include <stdio.h>


int   main( int argc, const char * argv[])
{
   NSEnumerator   *rover;
   NSArray        *contents;

   rover    = [[NSFileManager defaultManager] enumeratorAtPath:@"demo2"];
   contents = [rover allObjects];
   contents = [contents sortedArrayUsingSelector:@selector( compare:)];
   printf( "%s\n", [[contents description] UTF8String]);

   return( 0);
}

