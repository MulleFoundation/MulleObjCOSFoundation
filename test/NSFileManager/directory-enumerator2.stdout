(
    a,
    "b/x",
    "b/y/o",
    "b/z",
    c
)
